swift/Picture.swift

```diff
@@ -0,0 +1,138 @@
+import CoreGraphics
+
+import thorvg
+
+/// A Swift wrapper for ThorVG's Picture, facilitating picture manipulation.
+class Picture {
+    /// Supported MIME types for loading picture data.
+    enum MimeType: String {
+        case svg = "svg"
+        case lottie = "lottie"
+        case jpg = "jpg"
+        case jpeg = "jpeg"
+        case svgxml = "svg+xml"
+        case png = "png"
+    }
+
+    /// Pointer to the underlying ThorVG picture object.
+    let pointer: OpaquePointer
+
+    /// Initializes a new Picture instance with an empty ThorVG picture object.
+    convenience init() {
+        let pointer: OpaquePointer = tvg_picture_new()
+        self.init(pointer: pointer)
+    }
+
+    /// Initializes a Picture instance from an existing ThorVG picture pointer.
+    init(pointer: OpaquePointer) {
+        self.pointer = pointer
+    }
+
+    /// Loads a picture from a given file path.
+    func load(fromPath path: String) throws {
+        guard tvg_picture_load(pointer, path) == TVG_RESULT_SUCCESS else {
+            throw ThorVGError.failedToLoadFromPath
+        }
+    }
+
+    /// Loads a picture from a given data string.
+    ///
+    /// Use the `mimeType` to indicate the data format for correct parsing.
+    func load(fromString string: String, mimeType: MimeType) throws {
+        guard let cString = string.cString(using: .utf8),
+              tvg_picture_load_data(pointer, cString, UInt32(cString.count), mimeType.rawValue, false) == TVG_RESULT_SUCCESS
+        else {
+            throw ThorVGError.failedToLoadFromDataString
+        }
+    }
+
+    /// Resizes the picture content to the given size.
+    func resize(_ size: CGSize) {
+        tvg_picture_set_size(pointer, Float(size.width), Float(size.height))
+    }
+
+    /// Retrieves the size of the picture.
+    private func getSize() -> CGSize {
+        var width: Float = 0
+        var height: Float = 0
+        tvg_picture_get_size(pointer, &width, &height)
+        return CGSize(width: Double(width), height: Double(height))
+    }
+
+    /// Applies a transformation matrix to the picture, with an optional anchor point for rotation and scaling.
+    ///
+    /// Note: the anchorPoint defaults to (0.5, 0.5), which is the center of the picture.
+    func apply(transform: CGAffineTransform, anchorPoint: CGPoint = CGPoint(x: 0.5, y: 0.5)) {
+        let size = getSize()
+
+        let pivotPoint = CGPoint(
+            x: size.width * anchorPoint.x,
+            y: size.height * anchorPoint.y
+        )
+
+        let transform = getTransform()
```

Comment:
hmm... 

you passed in `transform` as a parameter in line 65, then you created local `let transform` at line 73

in this case, I think you never actually use the `transform` at all from line 65 ???